src/inference/include/ie/ie_core.hpp

```diff
@@ -110,6 +123,19 @@ class INFERENCE_ENGINE_API_CLASS(Core) {
                                   const std::string& deviceName,
                                   const std::map<std::string, std::string>& config = {});
 
+    /**
+     * @brief Reads model and creates an executable network from IR or ONNX file and uses AUTO plugin as the default
+     * device to load executable network.
+     *
+     * This can be more efficient than using ReadNetwork + LoadNetwork(CNNNetwork) flow
+     *        especially for cases when caching is enabled and cached model is available
+     *
+     * @param modelPath path to model
+     *
+     * @return An executable network reference
+     */
+    ExecutableNetwork LoadNetwork(const std::string& modelPath);
```

Comment:
why not just add a default value of "AUTO" for the deviceName in the below (original method)... your version doesn't allow to set config btw

Reply:
also where is OV 2.0 API equivalent (compile_network) ?
adding @ilya-lavrenov

Reply:
As you mentioned in the previous PR https://github.com/openvinotoolkit/openvino/pull/8494#discussion_r757304345, we defined this kind of LoadNetwork version in order to distinguish whether the user passed "AUTO" or used the default value of "AUTO" for deviceName.